### PR TITLE
Ds custom pk query update

### PR DESCRIPTION
**Pull request by david-mcafee**
#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-js/blob/main/CONTRIBUTING.md#steps-towards-contributions)
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by dpilch — APPROVED** [review]

File: packages/datastore/src/util.ts [hunk 1]
```diff
@@ -36,6 +36,8 @@ const ID = 'id';
 
 export const errorMessages = {
 	idEmptyString: 'An index field cannot contain an empty string value',
+	queryByPkWithCompositeKeyPresent:
+		'Cannot query by primary key exclusively when a composite key is present',
```
Comment (iartemiev) [inline packages/datastore/src/util.ts:40]: I would say:

Models with composite primary keys cannot by queried by a single field. Use [link to docs OL example]

Comment (david-mcafee) [reply packages/datastore/src/util.ts:40]: You are a wordsmith

Comment (david-mcafee) [reply packages/datastore/src/util.ts:40]: You are a wordsmith

Comment (david-mcafee) [reply packages/datastore/src/util.ts:40]: Given the deep level of nesting in our CPK docs, I'll have to link to the parent section, but I'll reference the OL subsection.

**Review by iartemiev — APPROVED** [review]
LGTM! 👍🏻

**Pull request merged**